test/platform/scenes/portable-experiences.spec.ts

```diff
@@ -0,0 +1,38 @@
+import expect from 'expect'
+import { PortableExperiences, PermissionItem } from '../../../src'
+
+describe('Portable experiences tests', () => {
+  const toggles: PortableExperiences = {
+    permission: PermissionItem.PI_ALLOW
+  }
+
+  it('type has a "schema" object', () => {
+    expect(typeof PortableExperiences.schema).toEqual('object')
+  })
+
+  it('type has a "validate" function', () => {
+    expect(typeof PortableExperiences.validate).toEqual('function')
+  })
+
+  it('static tests must pass', () => {
+    expect(PortableExperiences.validate(toggles)).toEqual(true)
+    expect(PortableExperiences.validate({})).toEqual(true)
+    expect(PortableExperiences.validate(null)).toEqual(false)
+  })
+
+  it('feature with number value fails', () => {
+    expect(PortableExperiences.validate({ permission: 1 })).toEqual(false)
```

Comment:
For all the cases in which the result is false, could you add some test for the error messages it generates? In general, the default messages from ajv are so bad that usually mean nothing to the user adding them. So in these cases we might want to add custom error messages. Not say this particular one is the case, but from looking at the message we'll figure this out.